interfaces/builtin/custom_device_test.go

```diff
@@ -83,6 +84,9 @@ slots:
       environment:
         env1: first
         env2: second|other
+    - kernel: msr[0-9]*
+      subsystem: msr
+      device: /dev/cpu/[0-9]*/msr
```

Comment:
there is a bit of an issue that this field doesn't work as any other fields, is not adding DEVICE== to the rule, but just making sure that the right KERNEL== is emitted? I wonder if a different name or approach would be clearer

Reply:
Per meeting with @pedronis, we'll change the attribute key to `for-device`, so that there's no confusion that there could be a udev rule which has a similar meaning but we still have a clear correspondence between the particular udev-tagging entry and one from `devices`